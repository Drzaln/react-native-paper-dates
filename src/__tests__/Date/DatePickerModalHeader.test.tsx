import { render } from '@testing-library/react-native'
import { SafeAreaProvider } from 'react-native-safe-area-context'
import DatePickerModalHeader from '../../Date/DatePickerModalHeader'

it('renders DatePickerModalHeader', () => {
  const { toJSON } = render(
    <SafeAreaProvider>
      <DatePickerModalHeader
        locale="en"
        onSave={() => null}
        onDismiss={() => null}
        saveLabel="Save"
      />
    </SafeAreaProvider>
  )
  expect(toJSON()).toMatchSnapshot()
})
