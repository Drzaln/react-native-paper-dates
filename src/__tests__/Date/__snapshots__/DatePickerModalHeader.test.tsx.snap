// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DatePickerModalHeader 1`] = `null`;
